src/server/middleware/logs.ts

```diff
@@ -1,71 +1,90 @@
-import type { FastifyInstance } from "fastify";
+import type { FastifyInstance, FastifyRequest } from "fastify";
 import { stringify } from "thirdweb/utils";
 import { logger } from "../../shared/utils/logger";
 import { ADMIN_QUEUES_BASEPATH } from "./admin-routes";
 import { OPENAPI_ROUTES } from "./open-api";
 
-const SKIP_LOG_PATHS = new Set([
-  "",
+const IGNORE_LOG_PATHS = new Set([
   "/",
   "/favicon.ico",
   "/system/health",
   "/static",
   ...OPENAPI_ROUTES,
-  // Skip these routes case of importing sensitive details.
+]);
+
+const SENSITIVE_LOG_PATHS = new Set([
   "/backend-wallet/import",
   "/configuration/wallets",
+  "/backend-wallet/lite",
 ]);
 
+function shouldLog(request: FastifyRequest) {
```

Comment:
Small changes here to add the `POST /backend-wallet/lite` path to not log the litePassword